pulp_rpm/app/serializers/comps.py

```diff
@@ -177,5 +177,13 @@ class CompsXmlSerializer(serializers.Serializer):
         write_only=True,
     )
 
+    # TODO: determine what comps is uploaded
```

Comment:
Same.  Should we mark it as draft again?

Reply:
No. This is left over from ppicka's POC. There is no "comps object", it's a file-uplaod that decomposes into multiple entities. At upload-time, there is no "already existing" comps to return. Removed the dead code.